### PR TITLE
Zsh completion: permit multiple --file arguments

### PR DESCRIPTION
**Pull request by mattjbray**
Before this change:

```
$ docker-compose --file docker-compose.yml -<TAB>
 -- option --
--help                 -h  -- Get help
--host                 -H  -- Daemon socket to connect to
--project-name         -p  -- Specify an alternate project name (default: directory name)
--skip-hostname-check      -- Don't check the daemon's hostname against the name specified in the client certificate (for example if your docker host is an IP address)
--tls                      -- Use TLS; implied by --tlsverify
--tlscacert                -- Trust certs signed only by this CA
--tlscert                  -- Path to TLS certificate file
--tlskey                   -- Path to TLS key file
--tlsverify                -- Use TLS and verify the remote
--verbose                  -- Show more output
--version              -v  -- Print version and exit
```

(Note the `--file` argument is no longer available to complete.)

After this change:

```
docker-compose --file docker-compose.yml -<TAB>
 -- option --
--file                 -f  -- Specify an alternate docker-compose file (default: docker-compose.yml)
--help                 -h  -- Get help
--host                 -H  -- Daemon socket to connect to
--project-name         -p  -- Specify an alternate project name (default: directory name)
--skip-hostname-check      -- Don't check the daemon's hostname against the name specified in the client certificate (for example if your docker host is an IP address)
--tls                      -- Use TLS; implied by --tlsverify
--tlscacert                -- Trust certs signed only by this CA
--tlscert                  -- Path to TLS certificate file
--tlskey                   -- Path to TLS key file
--tlsverify                -- Use TLS and verify the remote
--verbose                  -- Show more output
--version              -v  -- Print version and exit
```

Signed-off-by: Matt Bray mattjbray@gmail.com

### CODE REVIEW

**Comment by aanand** [comment]
Nice. The description is a bit wrong (it's not an "alternate" file when you specify the second file, or the third, etc) but I'm not sure how to reword it to be both accurate and non-confusing.

**Comment by mattjbray** [comment]
Makes sense. This is the new behaviour:

```
$ docker-compose -<TAB>
 -- option --
--file                 -f  -- Specify an alternate docker-compose file (default: docker-compose.yml
--help                 -h  -- Get help
...
$ docker-compose -f docker-compose.yml -<TAB>
 -- option --
--file                 -f  -- Specify an override docker-compose file (default: docker-compose.override.yml)
--help                 -h  -- Get help
...
```

What do you think?

**Comment by shin-** [comment]
LGTM - Thank you!
